src/types/ScreenInfoUiaProvider.cpp

```diff
@@ -31,9 +31,22 @@ SAFEARRAY* BuildIntSafeArray(_In_reads_(length) const int* const data, const int
     return psa;
 }
 
+ScreenInfoUiaProvider::ScreenInfoUiaProvider(_In_ Microsoft::Console::Render::IRenderData* pData,
+                                             _In_ WindowUiaProviderBase* const pUiaParent,
+                                             _In_ std::function<RECT(void)> GetBoundingRect) :
+    _pUiaParent(pUiaParent), //THROW_HR_IF_NULL(E_INVALIDARG, pUiaParent)),

```

Comment:
Get rid of commented out THROW or reintroduce it? (Here and below)

Reply:
Added it to GitHub #2120